tools-local/init-build.proj

```diff
@@ -155,7 +155,7 @@
 
   <Target Name="BuildDarc" Condition="'$(OfflineBuild)' != 'true'">
```

Comment:
Minor: Rename to `AcquireDarc` or similar?

Reply:
That would make it more clear 👍